### DISCUSSION TITLE
[bug] [PL] Interfejs pokazuje LBL zamiast poprawnych nazw

### DISCUSSION THREAD

**Discussion by cedeka78** [YetiForceCompany/YetiForceCRM#15750] [General]
Wersja 6.3.0
Problem występuje zarówno w wersji użytkowanej, jak i świeżo zainstalowanej w celu potwierdzenia występowania problemu.

Interfejs nie pokazuje poprawnych nazw tylko np.: LBL_EVENTS
![lbl](https://user-images.githubusercontent.com/15181660/155967672-1d8a369f-9c36-4d72-9d4c-56b8d0628282.png)

Problem rozwiązuje się jeśli w konfiguracji systemu ustawimy na "Tak" opcję "Pokaż wybór języka na stronie logowania"
![language_selection](https://user-images.githubusercontent.com/15181660/155969078-5ae8898a-40ac-4d7c-836d-8f6274638ad2.png)

Wówczas to samo okno kalendarza wygląda już poprawnie
![calendar](https://user-images.githubusercontent.com/15181660/155969688-dfa7ad8c-71dd-4d84-85d2-7300f1c21e62.png)

**Comment by mariuszkrzaczkowski** [comment]
załącz zrzut z ustawień użytkownika, sekcja  "Układ graficzny"
![image](https://user-images.githubusercontent.com/10198654/155972132-fd48df8f-817d-4d7f-b900-b4e2b6ce78df.png)

**Comment by cedeka78** [comment]
Okazuje się, że po wyborze i zapisaniu w ustawieniach użytkownika jakiegokolwiek języka i tak ten wybór znika, jakby nie zostaje zapisany
![układ_graficzny](https://user-images.githubusercontent.com/15181660/155974153-1a5ce4a8-13de-40fb-8d4d-5956f1f30e27.png)

**Reply by mariuszkrzaczkowski** [reply]
Tu jest problem, brak języka

**Reply by mariuszkrzaczkowski** [reply]
![image](https://user-images.githubusercontent.com/10198654/155979084-94685b28-fc9d-473a-b67c-e64fff7674d9.png)

**Reply by cedeka78** [reply]
A z czego może wynikać, że tego języka brakuje? Język mogę wybrać, ale po zapisaniu ten wybór jakby znika.

**Reply by mariuszkrzaczkowski** [reply]
To problem z konfiguracją serwera, proszę o załączenie raportu z konfiguracji serwera https://yetiforce.com/pl/baza-wiedzy/dokumentacja/dokumentacja-administratora/item/logi-serwer-konfiguracja

**Reply by cedeka78** [reply]
załączam
![ConfReport](https://user-images.githubusercontent.com/15181660/155982279-f1bd9a4c-639a-4c39-b7ba-83d2d2ac928f.png)

**Reply by mariuszkrzaczkowski** [reply]
To co na czerwono należy poprawić i problem z zapisem zostanie naprawiony

**Reply by cedeka78** [reply]
A dostałbym jakąś wskazówkę od czego zacząć naprawianie w przypadku tego błędu? Trochę tych czerwonych pól jest. Od lat korzystamy z serwera na home.pl pod YetiForce i dosyć poprawnie to funkcjonuje, ale nie wiem czy wszystkie wymagane ustawienia uda się poprawić.

**Reply by mariuszkrzaczkowski** [reply]
Błędów jest masa ale ich nie widać bo są tłumione, dopiero to będzie widać tak jak z językiem czegoś będzie brakować , coś się nie wgra na serwer dokumenty będą znikać , dane będą znikać itp.
Ten panel powstał na bazie napotkanych problemów od wielu lat więc wszystko należy poprawić inaczej będą błędy ujawniać się stopniowo,

**Reply by cedeka78** [reply]
Rozumiem. Dziękuję za pomoc
